src/Microsoft.SqlTools.ServiceLayer/QueryExecution/DataStorage/SaveAsExcelFileStreamWriterHelper.cs

```diff
@@ -0,0 +1,688 @@
+﻿
```

Comment:
Understand. 
I wrote all the code freshly, thus there should not be any licence problem.

All the code are written based on the ECMA-376, with some ideas from an unzipped empty excel generated xlsx file. I never reverse-engineering the Excel.

Background: I was looking for for a good library for exporting query result for dotnet core, but can't find one. One day, I opened up a xlsx file and found it is not that complex. I think I can create one within 300 lines. Thus, this code.

I have a question, may I reuse some of these code and make a general export to excel library later? I will licence the library as MIT.